lab-bundle-spec.md

```diff
@@ -141,15 +141,10 @@ id        |          | string | Identifier that can be used throughout project b
 environment_resources:
   - type: gcp_project
     id: my_primary_project
-    dm_template:
-      script: deployment_manager
+  - type: gcp_project
+    id: secondary_project
   - type: gcp_user
     id: primary_user
-    permissions:
```

Comment:
what was the motivation behind removing this section?

Reply:
I felt like it was weird to have this example include permissions but not any of the other resource-specific stuff defined below. Since this is the example right below the type/id definition, I thought it made sense for it to just have types and ids.